moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -129,8 +127,7 @@ void operator>>(const YAML::Node& node, T& i)
 typedef const YAML::Node* yaml_node_t;
 
 // Helper function to find a value (yaml-cpp 0.3)
```

Comment:
can we just remove support for yaml-cpp 0.3 at this point?

Reply:
yaml-cpp 0.3 is default for Ubuntu 14.04 / ROS Indigo. In the Kinetic branch, indeed we could get rid of 0.3 support.

Reply:
thanks!